src/main/steps/applicant1/hub-page/template.njk

```diff
@@ -16,7 +16,7 @@
       <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible govuk-section-break--thinner">
```

Comment:
```suggestion
      <hr class="govuk-section-break govuk-section-break--l govuk-section-break--additional">
```
These classes make the top divider in the hub page blue to match the design:
<img width="783" alt="image" src="https://github.com/user-attachments/assets/adbfce26-be18-4cf7-91a0-ead583728fb7">